examples/cpp/pyperf/PyPerfBPFProgram.cc

```diff
@@ -313,6 +317,47 @@ on_event(struct pt_regs* ctx) {
   event->stack_status = STACK_STATUS_ERROR;
   event->error_code = ERROR_NONE;
 
+  struct task_struct const *const task = (struct task_struct *)bpf_get_current_task();
+
+  // Get raw native user stack
```

Comment:
Can put this whole thing in an `if (sizeof(event->raw_user_stack) > 0)` so it gets optimized out and doesn't run if I pass `--user-stack-pages 0`